opslevel/resource_opslevel_check_custom_event.go

```diff
@@ -22,6 +22,12 @@ func resourceCheckCustomEvent() *schema.Resource {
 				ForceNew:    false,
 				Required:    true,
 			},
+			"pass_pending": {
+				Type:        schema.TypeBool,
+				Description: "True if this check should pass by default. Otherwise the default 'pending' state counts as a failure.",
+				ForceNew:    false,
+				Optional:    true,
```

Comment:
Since this is expected to be "Non Null" in the API and line 58 will be setting it every time into the payload.  We should make it Required for Terraform.